VALUES.md

```diff
@@ -0,0 +1,30 @@
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at creating a new foundation of project governance and management that brings **humans** and their concerns front and center.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations, package management, libuv, v8 and Chakra, builds/ops, testing and its infrastructure, hardware and IoT, Android, streams, community events, facilitation, moderation and education. These and more are all concerns that can be handled separately and by members who might be much better qualified for their individual tasks than a central decision body.
+
+Ayo.js wants to go further than Node Core did in spreading responsibility across the project, so everyone can focus and specialize better than before. Contributors should be able to organize and manage their own subprojects in a way that is effective and efficient for their individual unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance policy should change with the required agility. Following the governance policy to the letter should not take precedence over staying true to our stated values, not to mention the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with members with all sorts of perspectives. You're bound to find yourself in situations with no clear consensus, with new concerns being brought up on a regular basis, and things that might seem insignificant in other contexts suddenly explode.
+
+The Ayo.js project's perspective is that when this happens, it often takes a toll on the well-being of the members participating, or it prevents potential members from contributing again, if at all. It also recognizes that the difference between individual choices is less important than the ability to come to a perfect technical decision.
+
+Projects change, evolve, and find themselves dealing with new concerns all the time. In all technical matters, Ayo.js will take into account that in the end, it's the people making those decisions that matter most, not the decisions themselves. It's ok for code to be imperfect.
+
+## Humans are more important than corporations.
+
+All sorts of corporations and other organizations around the world use and benefit from the software underlying Ayo.js. Many contributors, in fact, might have their efforts in the project directly funded by them. But, this can often turn into such direct reliance on corporations that they can essentially seize control of a project -- or struggle for power over it while individual contributors are left wondering if they can ever influence the project themselves.
```

Comment:
Turning this into a more general statement (about all open source software) would be a strong statement!

(The next paragraph emphasizes Ayo.js's approach, so this paragraph doesn't need to be Ayo.js-specific.)